src/key_events.h

```diff
@@ -43,6 +43,30 @@ extern const Key keymaps[][ROWS][COLS];
  */
 void handleKeyswitchEvent(Key mappedKey, byte row, byte col, uint8_t keyState);
 
-// Internal use
+/** Flushes any pending regular key switch events and sends them out */
+void sendKeyboardReport();
+
+// A facade on top of our HID implementation
 void pressKey(Key mappedKey);
 void releaseKey(Key mappedKey);
+void releaseAllKeys();
+void pressKeyRaw(Key mappedKey);
+void releaseKeyRaw(Key mappedKey);
+
+void pressConsumer(Key mappedKey);
+void releaseConsumer(Key mappedKey);
+
```

Comment:
Reading this a second time, could I suggest `pressConsumerKey` instead? Just to make it crystal clear that we are not trying to press the user. :wink: